.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### What happened?
+    Tell us what happened and provide as much details as possible, including logs.
+
+### What did you expect to happen?
+    Tell us about expected behaviour.
+
+### What is the GPU model?
+    Tell us about hardware configuration of the GPU.
+
+### What is the environment?
+    Is the DCGM exporter bare-metal or cloud provider? Do you run the DCGM exporter on Kubernetes or maybe as a standalone Docker container?
```

Comment:
"Is DCGM-Exporter running on bare metal or in a virtual environment, container, pod, etc?"